limit.go

```diff
@@ -0,0 +1,75 @@
+package pilosa
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"time"
+)
+
+var (
+	ErrorTooManyConnections = errTooManyConnections{}
+)
+
+func NewBoundListener(maxActive int, l net.Listener) net.Listener {
+	b := &boundListener{l, make(chan struct{}, maxActive)}
+	for i := 0; i < maxActive; i++ {
+		b.active <- struct{}{}
+	}
+	return b
+}
+
+type boundListener struct {
+	net.Listener
+	active chan struct{}
+}
+
+type boundConn struct {
+	net.Conn
+	active chan struct{}
+}
+type errTooManyConnections struct {
+}
+
+func (e errTooManyConnections) Error() string   { return "To Many Connections" }
```

Comment:
Typo in "Too".
Also, I don't think `Many Connections` should be capitalized.
i.e. `Too many connections`